showcase/src/stark-app-config.json

```diff
@@ -10,7 +10,7 @@
   "sessionTimeoutWarningPeriod": 20,
   "keepAliveUrl": "http://localhost:5000/keepalive",
   "keepAliveInterval": 20,
-  "keepAliveDisabled": true,
+  "keepAliveDisabled": false,
```

Comment:
Could we remove this property instead of setting it as `false` ? 😊

Reply:
indeed